python/interpret_community/widget/explanation_dashboard.py

```diff
@@ -7,7 +7,7 @@
 import os
 import json
 from .explanation_dashboard_input import ExplanationDashboardInput
-from ._internal import DatabricksInterfaceConstants
+from ._internal.constants import DatabricksInterfaceConstants
```

Comment:
@rihorn2 we need some testing at least that can catch import errors

Reply:
Yes, I agree

Reply:
will work on adding some automated notebook testing